content/blog/2016/2016-09-30-jenkins-world-2016-wrap-up-community.adoc

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: "Jenkins World 2016 Wrap-up - Community"
```

Comment:
My vote would be for the last one, it's got a cheekiness that I enjoy but might not be inline with your voice